Neos.Flow/Classes/ObjectManagement/Configuration/ConfigurationBuilder.php

```diff
@@ -292,18 +297,14 @@ protected function parseConfigurationArray($objectName, array $rawConfigurationO
      * @return integer The scope translated into a Configuration::SCOPE_* constant
      * @throws InvalidObjectConfigurationException if an invalid scope has been specified
      */
-    protected function parseScope($value)
+    protected function parseScope(string $value): int
```

Comment:
for all `parse*` functions it might make sense to prefer mixed as argument and throwing a real exception than risking a php type error - what do you say?